angular-date-picker.js

```diff
@@ -15,26 +15,51 @@
 }(this, function (angular) {
     'use strict';
 
-    return angular.module('mp.datePicker', []).directive('datePicker', [ '$window', function ($window) {
+    return angular.module('mp.datePicker', []).directive('datePicker', [ '$window', '$locale', function ($window, $locale) {
         // Introduce custom elements for IE8
         $window.document.createElement('date-picker');
 
         var tmpl = ''
 + '<div class="angular-date-picker">'
 + '    <div class="_month">'
-+ '        <button type="button" class="_previous" title="Previous Month" ng-click="changeMonthBy(-1)">&laquo;</button>'
-+ '        {{ months[month].shortName }} {{ year }}'
-+ '        <button type="button" class="_next" title="Next Month" ng-click="changeMonthBy(1)">&raquo;</button>'
++ '        <button type="button" class="_previous" ng-click="changeMonthBy(-1)">&laquo;</button>'
++ '        <span title="{{ months[month].fullName }}">{{ months[month].shortName }}</span> {{ year }}'
++ '        <button type="button" class="_next" ng-click="changeMonthBy(1)">&raquo;</button>'
 + '    </div>'
 + '    <div class="_days" ng-click="pickDay($event)">'
-+ '        <div class="_day-of-week" ng-repeat="dayOfWeek in daysOfWeek">{{ dayOfWeek.letter }}</div>'
++ '        <div class="_day-of-week" ng-repeat="dayOfWeek in daysOfWeek" title="{{ dayOfWeek.fullName }}">{{ dayOfWeek.firstLetter }}</div>'
 + '        <div class="_day -padding" ng-repeat="day in leadingDays">{{ day }}</div>'
-+ '        <div class="_day -selectable" ng-repeat="day in days" ng-class="{ \'-selected\': (day === selectedDay), \'-today\': (day === today) }" title="{{ day === today ? \'Today\' : \'\' }}">{{ day }}</div>'
++ '        <div class="_day -selectable" ng-repeat="day in days" ng-class="{ \'-selected\': (day === selectedDay), \'-today\': (day === today) }">{{ day }}</div>'
 + '        <div class="_day -padding" ng-repeat="day in trailingDays">{{ day }}</div>'
 + '    </div>'
 + '</div>'
         ;
 
+        var days = [], // Slices of this are used for ngRepeat
```

Comment:
Since even Angular's own date filter doesn't do that currently, it wouldn't add much value here. `$locale` changes, it seems, don't get picked up dynamically elsewhere.